### ISSUE TITLE
[Adaptive Card iOS - Input.text.json]: 'Default text input' label is not associated with its edit box.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7337]
https://microsoft.visualstudio.com/OS/_workitems/edit/37830074
